### PR TITLE
Lu1 41 assignment develop the end points along with error handling

### PR DESCRIPTION
**Pull request by Dhanrajnath**
Hi @sats26 ,
please review this PR regarding developing REST APIs for the given problem statement, I've updated the README file with Postman UI screenshots of API requests and responses.
Thank you!

### CODE REVIEW

File: DB Setup Seeder/app.js [hunk 1]
```diff
@@ -3,42 +3,49 @@ const path = require("path");
 const express = require("express");
 const bodyParser = require("body-parser");
 const sequelize = require("./util/database");
-const User = require("./models/user");
 const Associations = require("./models/associations")();
 
+const userRoutes = require("./routes/user");
```
Comment (sats26) [inline DB Setup Seeder/app.js]: don't have these many routes here. have it in a common router and import these there.

File: DB Setup Seeder/app.js [hunk 2]
```diff
@@ -3,42 +3,49 @@ const path = require("path");
...
+    });
+});
+
+app.use(userRoutes);
+app.use(paymentRoutes);
+app.use(contractRoutes);
+app.use(cashkickRoutes);
+
+app.use((error, req, res, next) => {
+  const status = error.statusCode || 500;
```
Comment (sats26) [inline DB Setup Seeder/app.js]: try to use es6 object destructing

File: DB Setup Seeder/controllers/contract.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      if (contracts.length === 0) {
+        const error = new Error("No contracts found!!");
+        error.statusCode = 200;
+        throw error;
+      }
+      res.status(200).json({ data: contracts });
+    })
+    .catch((err) => {
+      if (!err.statusCode) {
+        err.statusCode = 500;
```
Comment (sats26) [inline DB Setup Seeder/controllers/contract.js:58]: handle common errors using middleware. lot of replicated error and success code across controller.

File: DB Setup Seeder/images/CASHKICKS/add_another_cashkick.jpg
Comment (sats26) [inline DB Setup Seeder/images/CASHKICKS/add_another_cashkick.jpg:1]: images are not mostly required. you can have postman collection export json here. over postman you can add examples for apis.

**Review by sats26 — CHANGES_REQUESTED** [review]

File: DB Setup Seeder/app.js (hunk 1 above)
Comment (Dhanrajnath) [reply DB Setup Seeder/app.js]: ok satheesh, will make a common router file

File: DB Setup Seeder/images/CASHKICKS/add_another_cashkick.jpg
Comment (Dhanrajnath) [reply DB Setup Seeder/images/CASHKICKS/add_another_cashkick.jpg:1]: ok Satheesh, will update.

**Review by sats26 — APPROVED** [review]

**Pull request merged**